workflow/tests/base_results/results_simulations_hvac.csv

```diff
@@ -138,17 +138,18 @@ base-enclosure-garage.xml,6.8,91.76,36000.0,24000.0,0.0,28677.0,6804.0,5506.0,0.
 base-enclosure-infil-ach-house-pressure.xml,6.8,91.76,36000.0,24000.0,0.0,32616.0,8677.0,7508.0,0.0,575.0,6840.0,0.0,0.0,2148.0,2171.0,4697.0,0.0,19910.0,6101.0,7037.0,0.0,207.0,321.0,0.0,0.0,0.0,2293.0,630.0,0.0,3320.0,0.0,133.0,0.0,-667.0,0.0,800.0
 base-enclosure-infil-cfm-house-pressure.xml,6.8,91.76,36000.0,24000.0,0.0,32618.0,8677.0,7508.0,0.0,575.0,6840.0,0.0,0.0,2148.0,2171.0,4699.0,0.0,19910.0,6101.0,7037.0,0.0,207.0,321.0,0.0,0.0,0.0,2293.0,631.0,0.0,3320.0,0.0,132.0,0.0,-668.0,0.0,800.0
 base-enclosure-infil-cfm50.xml,6.8,91.76,36000.0,24000.0,0.0,32618.0,8677.0,7508.0,0.0,575.0,6840.0,0.0,0.0,2148.0,2171.0,4699.0,0.0,19910.0,6101.0,7037.0,0.0,207.0,321.0,0.0,0.0,0.0,2293.0,631.0,0.0,3320.0,0.0,132.0,0.0,-668.0,0.0,800.0
-base-enclosure-infil-ela.xml,6.8,91.76,36000.0,24000.0,0.0,37830.0,8798.0,7508.0,0.0,575.0,6840.0,0.0,0.0,2148.0,2171.0,9790.0,0.0,20619.0,6126.0,7037.0,0.0,207.0,321.0,0.0,0.0,0.0,2293.0,1314.0,0.0,3320.0,0.0,0.0,0.0,0.0,0.0,0.0
+base-enclosure-infil-ela.xml,6.8,91.76,36000.0,24000.0,0.0,37830.0,8798.0,7508.0,0.0,575.0,6840.0,0.0,0.0,2148.0,2171.0,9790.0,0.0,20565.0,6072.0,7037.0,0.0,207.0,321.0,0.0,0.0,0.0,2293.0,1314.0,0.0,3320.0,0.0,-591.0,0.0,-1391.0,0.0,800.0
```

Comment:
Note that some of these diffs are because we used to prevent a negative value for latent cooling loads. It's been this way since the initial implementation of MJ8. But I have confirmed with another ACCA accredited software tool that negative values are fine; you can report a negative latent cooling load and the total cooling load can indeed be lowered by a negative latent cooling load.